docs/topics/sessions.rst

```diff
@@ -24,16 +24,19 @@ in your ``asgi.py``:
 .. code-block:: python
 
     from channels.routing import ProtocolTypeRouter, URLRouter
+    from channels.security.websocket import AllowedHostsOriginValidator
     from channels.sessions import SessionMiddlewareStack
 
     from myapp import consumers
 
     application = ProtocolTypeRouter({
 
-        "websocket": SessionMiddlewareStack(
-            URLRouter([
-                url(r"^front(end)/$", consumers.AsyncChatConsumer.as_asgi()),
-            ])
+        "websocket": AllowedHostsOriginValidator(
+            SessionMiddlewareStack(
+                URLRouter([
+                    url(r"^front(end)/$", consumers.AsyncChatConsumer.as_asgi()),
```

Comment:
`url` is deprecated/removed from Django?

Maybe worth updating to use `path()`?

Reply:
Ok here's a separate PR for removing url(): #1826